apps/chat-e2e/src/tests/chatSelectionFunctionality.test.ts

```diff
@@ -213,7 +213,7 @@ dialTest(
 
     await dialTest.step('Click on Duplicate', async () => {
       await conversations.openEntityDropdownMenu(firstConversation.name, 2);
-      await conversationDropdownMenu.selectMenuOption(MenuOptions.duplicate);
+      await conversations.selectDuplicateMenuOption();
```

Comment:
We should indicate that we are waiting for the request result inside of the method and that is not 100% equal to await conversations.selectEntityMenuOption(MenuOptions.duplicate)

Reply:
The .selectEntityMenuOption() method no longer exists